doc/source/reference/server_variables.rst

```diff
@@ -471,6 +471,33 @@ value.
 
 TODO:
 
+.. _server-variable-mroonga-operations-disable-recording:
+
+``mroonga_operations_disable_recording``
```

Comment:
Inverted variable name confuses users.
Please use `mroonga_operations_enable_recording` and the default value is `ON`.

By the way, how about `mroonga_enable_operations_recording`? Because there is already `mroonga_enable_XXX` variable such as `mroonga_enable_optimization`.